tools/c7n_azure/c7n_azure/resources/cosmos_db.py

```diff
@@ -151,7 +102,10 @@ def _query_rules(self, resource):
             else:
                 return IPSet(['0.0.0.0/0'])
 
-        parts = ip_range_string.split(',')
+        parts = ip_range_string.replace(' ', '').split(',')
+
+        # Exclude magic strings representing Portal and Azure Cloud
+        parts = list(set(parts) - set(PORTAL_IPS + AZURE_CLOUD_IPS))
```

Comment:
minor:  I think we should do a contains check on the portal IP set and then remove it.

Only the total set represents the bypass (and the portal checkbox) - but if a user just manually added a subset of the portal set then it would be incorrect to remove them here.